src/elements/element.line.js

```diff
@@ -145,35 +363,127 @@ class Line extends Element {
 		super(props);
 	}
 
-	draw(ctx) {
+	set points(points) {
+		this._points = points;
+		delete this._segments;
+	}
+
+	get points() {
+		return this._points;
+	}
+
+	get segments() {
+		return this._segments || (this._segments = computeSegments(this));
```

Comment:
I think we always need to reference `_segments`? If that's true then could just compute it in the `constructor` instead?

Reply:
`points` are assigned after construntion.

Reply:
Also the point.skip is assigned after the points are assigned to line, so can't do it there either.